### ISSUE TITLE
Readme.md needs updating

### ISSUE THREAD

**Issue by MooersLab** [emacs-grammarly/send-to-osx-grammarly#2]
It seems that the path in the second code block needs to include the scritps folder.
Also, doesn't the push.scpt file need to be included?

**Comment by jcs090218** [comment]
Yes, the script folder would need to carry in to the package install location!

I have added to my own elpa [here](https://github.com/jcs-emacs/jcs-elpa), you can use it if you want to. Install through ELPA should take care of the that.
